inc/namespace.php

```diff
@@ -37,6 +37,10 @@ function setup() {
  * Schedule common maintenance tasks.
  */
 function schedule_events() {
+	if ( defined( 'WP_INSTALLING' ) && WP_INSTALLING ) {
```

Comment:
Just to double check, should this be `WP_INITIAL_INSTALL` instead?

Reply:
That only exists with Altis core so this is fine here. But now I think about it this might be getting added for every site. May need a follow up. Not sure if there's a solid solution for network level cron.

Reply:
Solution is to check if you're on the main site and only schedule there.